internal/service/workers/formsender/main.go

```diff
@@ -28,6 +30,22 @@ func Run(ctx context.Context, cfg config.Config) {
 			return nil
 		}
 
+		for i := range forms {
```

Comment:
```suggestion
		for i, f := range forms {
```
you are using twice for reading, why not to apply second iterator